.shopify-build/krane.yml

```diff
@@ -1,6 +1,6 @@
 containers:
   default:
-    docker: circleci/ruby:2.4.6-node-browsers
+    docker: circleci/ruby:2.5.7-node-browsers
```

Comment:
I wonder why we're using the `-node-browsers` image, especially since we're running our tests in their own container anyway.

Reply:
probably copy/paste from a doc that assumes rails.